misc.json

```diff
@@ -1134,6 +1134,17 @@
         "description": "App Spotlight\nMakes you discover new apps from the Google Play store. The selection criteria is unknown.",
         "removal": "delete"
     },
+    {
+        "id": "com.pivotmobile.android.metrics",
+        "label": "com.pivotmobile.android.metrics",
+        "description": "Pivot Mobile is a popular game developer famously known for its Swimmy Turtle game.\nUsed for data consumption & ad tracking of third-party apps, in some cases silently installs apps without your permission.\nCommonly found in Lenovo & Motorola devices.\nNot a system app.",
```

Comment:
What do you mean by “Not a system app”? If it's not a system app, how can it install apps without user interaction? White-listed by the system?